src/Server.ts

```diff
@@ -27,15 +30,15 @@ export default class Server {
 
   public setupRoute() {
     const { app } = this;
+    const jsonParser = bodyParser.json();
 
     app.use('/health-check', (req, res) => {
       res.status(200).send('I am OK');
     });
-
+    app.use('/api', jsonParser, router);
     app.use(notFoundRoute);
```

Comment:
No need to use parser here. In bootstrap function call initBodyParser () first.

Reply:
Do create all crud route for user as well

Reply:
routes created for users and fixed initBodyParser().

Reply:
You haven't done any changes